rs/tests/testnets/BUILD.bazel

```
@@ -279,6 +279,25 @@ system_test_nns(
     ],
 )
 
+system_test_nns(
+    name = "large_high_perf",
```

What about making it explicit this is for testing **I/O** performance by renaming the test to:
```suggestion
    name = "large_high_io_perf",
```